### ISSUE TITLE
access token not work

### ISSUE THREAD

**Issue by demohi** [Justineo/github-hovercard#11]
```
API limit exceeded API rate limit exceeded for current IP. Create a new access token and paste it back here to get a higher rate limit.
```

After I set the access token,  this error  still exist

**Comment by Justineo** [comment]
Is there any console error? And can you have a look at `localStorage.getItem('hovercard-token')` on GitHub? Is it set?

**Comment by drewbailey** [comment]
Having this issue as well.
```js
localStorage.getItem('hovercard-token')
null
```

**Comment by demohi** [comment]
same with @drewbailey

**Comment by Justineo** [comment]
@demohi @drewbailey What browser are you using? If you are using Chrome, would you plz check chrome://settings/content and what option is selected under "Cookies" section?

**Comment by twidi** [comment]
Note that I had the same problem. I did a check with `localStorage.getItem('hovercard-token')` and I saw my token prefixed with a space, added by double-clicking on the token on Github to copy it.

**Comment by Justineo** [comment]
@twidi it looks like your problem is not the same and providing invalid tokens will get a different error message. I can trim the token first to prevent that.

**Comment by demohi** [comment]
@JamieSinn  I am using Chrome Version 47.0.2526.106 (64-bit). Cookies is allowed to set.

**Comment by JamieSinn** [comment]
@demohi You probably mean @Justineo

**Comment by demohi** [comment]
@JamieSinn  sorry for that :disappointed_relieved:

**Comment by Justineo** [comment]
@demohi I sent a Weibo private message to you and perhaps we can discuss the problem more efficiently there :)

**Comment by Justineo** [comment]
Thanks to @demohi, the problem is probably caused by writing over 5MB into localStorage which exceeds its quota.

The upcoming solution might be:
1. Give an error message when failed to write into localStorage and let users to clear up
2. Use Chrome's option to store access tokens (should be the better way)

The fix will probably come with v0.4.

**Comment by Justineo** [comment]
Still using localStorage to save the access token because I haven't come up with a proper design for the options page if each domain needs an access token. The problem is described in project FAQ and I'm not going to change the logic for now.
